docs/pages/auto-discovery/databases.mdx

```diff
@@ -1,183 +1,219 @@
 ---
-title: AWS Database Auto-Discovery
-description: How to configure Teleport to discover AWS-hosted databases.
+title: Database Discovery
+description: Detailed guides for configuring database discovery.
 ---
 
-Database discovery involves two components.
+Teleport can be configured to discover databases automatically and register
+them with your Teleport cluster.
 
-The first, the Discovery Service, is responsible for watching your cloud
-provider and checking if there are any new databases or if there have been any
-modifications to previously discovered databases.
+## Supported clouds
 
-The second, the Database Service, monitors the databases created by the
-Discovery Service and proxies communications between users and the database
-servers hosted by the cloud provider.
+- [AWS](./databases/aws.mdx): Discovery for AWS databases.
+{/* TODO(gavin): Add an Azure discovery guide and permission reference */}
 
-In this guide, you will configure a Teleport Discovery Service instance to
-discover AWS-hosted databases automatically.
+## How database discovery works
 
-## Prerequisites
+Database auto-discovery consists of the following components:
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+1. One or more Teleport Discovery Services polling APIs to discover and register
```

Comment:
I've been working on standardizing the docs around "[Name] Service instance" instead of "[Name] Services", as each process runs a single service that can be configured in different ways.